misc/formatter/modules/formatter-core/src/test/resources/statements/call/assert/call_statement_2.bal

```diff
@@ -0,0 +1,13 @@
+import ballerina/io;
+
+function calculateTotalInvoiceAmount(int customerId, string invoiceDate,
+        decimal[] itemPrices, boolean isTaxable) returns decimal {
+    decimal totalAmount = 0.0;
+    return totalAmount;
+}
+
+public function main() {
+    decimal invoiceAmount = calculateTotalInvoiceAmount(12345, "2023-09-13",
+            [25.99, 19.95, 12.49, 7.99, 34.50], true);
```

Comment:
Shall we add a test for a nested multiline function call scenario if it is possible?